### ISSUE TITLE
Will there be an update to ITP in the future,？seat selection is really a big problem.

### ISSUE THREAD

**Issue by svtjoshuji** [crypt0nX/Interpark-Bot#2]
Will there be an update to ITP in the future, seat selection is really a big problem. Since the code for selecting a seat cannot be run out, even if you use a web page without a program to check the ticketed seats, all seats are gray out when using the program. The program that fills in the information doesn't seem to be able to run. This program written by the big guy is really good.,But I've changed a lot of parts now.,It's a little bit of the original program you wrote hahahahaha.,Because now it's not much with audio verification.,So I deleted that part.。 Selecting and filling in information is really a problem

**Comment by svtjoshuji** [comment]
Since Selemiun is no longer available on the website, many of the previously published international versions of ITP codes are no longer available. And I'm just an ordinary fan, because I don't want to buy high-priced second-hand tickets, so I do the program myself. I have used many programs, but they are basically half-finished or cannot be used. Big guy, this is the best to use, and the best to change, and it's also the best for me to understand.

**Comment by crypt0nX** [comment]
The seat selection part is the most difficult part of the whole program, because different events have different venues and I don't have the energy to adapt them one by one. As far as I know, some events don't seem to have a seat selection process. This code is just an idea as I said in the readme. It roughly points out the direction, but you need to implement which event and seat to choose by yourself.

For example, if you want to buy tickets for Stadium A, you need to search for all events at Stadium A on ITP, use them as an example to complete your seat selection code, and then change the necessary information in config to apply to your own event.

**Comment by svtjoshuji** [comment]
> The seat selection part is the most difficult part of the whole program, because different events have different venues and I don't have the energy to adapt them one by one. As far as I know, some events don't seem to have a seat selection process. This code is just an idea as I said in the readme. It roughly points out the direction, but you need to implement which event and seat to choose by yourself.
> 
> For example, if you want to buy tickets for Stadium A, you need to search for all events at Stadium A on ITP, use them as an example to complete your seat selection code, and then change the necessary information in config to apply to your own event.

That's right, seat selection is the hardest part, because I didn't find the location of the seat that can be modified in the program you posted at the beginning, so I changed the program directly, and I was studying it carefully, and we all paid a lot of time and energy for this seat selection hahahahaha, it's really hard. I used to use JS and Chrome browser plugins to implement seat selection, I could expand each area and see if there were any votes and then check it, but I don't mix JS and Python. Hahaha. I really had limited skills and didn't learn very deeply.

**Comment by svtjoshuji** [comment]
> The seat selection part is the most difficult part of the whole program, because different events have different venues and I don't have the energy to adapt them one by one. As far as I know, some events don't seem to have a seat selection process. This code is just an idea as I said in the readme. It roughly points out the direction, but you need to implement which event and seat to choose by yourself.
> 
> For example, if you want to buy tickets for Stadium A, you need to search for all events at Stadium A on ITP, use them as an example to complete your seat selection code, and then change the necessary information in config to apply to your own event.

Thank you very much for your prompt reply, which bothered you a lot of time. At the same time, I hope I can modify the effect I want. Hahaha
